### DISCUSSION TITLE
Где можно оставить/ поддержать запрос на тему?

### DISCUSSION THREAD

**Discussion by spokik** [doka-guide/content#2134] [Q&A] [answered]
Услышал про Доку в подкасте. Посмотрел, прикольно. Но когда мне понадобился материал, не нашел.

Так же не нашел где можно запросить, показать свой интерес к конкретному инструменту.
Я в этих гитхабах да и вообще в разработке не особо силен, большая часть написанного даже в дискуссиях для меня - тёмный лес. Возможно, если что-то подобное уже существует, стоит сделать более доступным, понятным
P.s.
Мне нужен был тернарный оператор JS для vue компонента, обычный if как-то не заходит

**Comment by pepelsbey** [comment, marked as answer]
Привет! Вы уже по адресу — в контентном репозитории. Если пофильтровать по лейблам [js + doka](https://github.com/doka-guide/content/issues?q=is%3Aopen+is%3Aissue+label%3Ajs+label%3A%D0%B4%D0%BE%D0%BA%D0%B0) в ишью, то можно найти материалы, которые хочется написать, часть из них уже в работе. Аналогичную ишью вы можете завести и сами. И там описать подробнее, чего вам не хватает, для какой задачи. К вам зайдут редакторы, расставят лейблы и расскажут, есть ли планы такое писать, попадает ли это в скоуп проекта.
